pkg/registry/rbac/clusterrolebinding/policybased/storage.go

```diff
@@ -63,6 +66,12 @@ func (s *Storage) Update(ctx genericapirequest.Context, name string, obj rest.Up
 	nonEscalatingInfo := rest.WrapUpdatedObjectInfo(obj, func(ctx genericapirequest.Context, obj runtime.Object, oldObj runtime.Object) (runtime.Object, error) {
 		clusterRoleBinding := obj.(*rbac.ClusterRoleBinding)
 
+		// if we're explicitly authorized to bind this clusterrole, return
+		if canBind, _, _ := rbacregistry.AuthorizeBinding(ctx, clusterRoleBinding.RoleRef, clusterRoleBinding.Namespace, s.authorizer); canBind {
```

Comment:
Is it worth doing something else (at least logging) if we failed to get an explicit "no" here?